tests/bugs/fuse/bug-858488-min-free-disk.t

```diff
@@ -69,7 +69,7 @@ done
 
 ## Bring free space on one of the bricks to less than minfree value by
 ## creating one big file.
-dd if=/dev/zero of=$M0/fillonebrick.data bs=1024 count=25600 1>/dev/null 2>&1
+dd if=/dev/zero of=$M0/fillonebrick.data bs=128k count=200 1>/dev/null 2>&1
```

Comment:
Not related to this patch, but I wonder if this is the best way to fill bricks. The size of the bricks could change for many reasons, so using a constant size doesn't seem the best option here. Also, an fallocate could be more efficient.